container-runtime/src/Adapters/Systems/SystemForEmbassy/polyfillEffects.ts

```diff
@@ -116,7 +116,7 @@ export class PolyfillEffects implements oet.Effects {
       this.manifest.volumes,
     )
     const daemon = dockerProcedureContainer.then((dockerProcedureContainer) =>
-      daemons.runDaemon()(
+      daemons.runCommand()(
```

Comment:
Why this change

Reply:
ok, as long as this `runCommand` isn't collecting output, we don't want to be leaking memory buffering logs